packages/react/src/exampleTypes/StepSlider.tsx

```diff
@@ -0,0 +1,186 @@
+  RangeSliderFilledTrack,
+  RangeSliderThumb,
+  Box,
+  Tooltip,
+  Text,
+  HStack,
+  Stack,
+} from '@chakra-ui/react'
+import { contexturify } from '../utils/hoc'
+import { toNumber } from '../utils/format'
+import { FormatterFn, Node, Tree } from '../types/contexture'
+
+/** This theme is an initial step towards looking further
+ * at the tokenization to be set by our design system. */
+const theme = {
+  color: {
+    blue: { 100: '#CCE7FF', 500: '#0076DE', 700: '#005199' },
+    gray: { 50: '#FFFFFF', 100: '#E7EAEE' },
+    container: { border: { sm: '#E7EAEE' } },
+    text: {
+      body: {
+        primary: '#495569',
+        tertiary: '#697996',
+      },
+    },
+  },
+  radius: { base: { xs: '8px' } },
+  font: { size: { xs: '10px', md: '14px' } },
+}
+
+export type UnknownRec = Record<string, unknown>
+
+type NodeProps = {
+  range: number[]
+  steps: number[]
+}
+
+type ContextProps = {
+  recordsCount: number
+}
+
+interface StepSliderProps {
+  node: Node<NodeProps, ContextProps>
+  formatter: FormatterFn
+  path: string
+  tree: Tree<NodeProps>
+}
+
+const getDefaultRange = (steps: number[], range: number[]): number[] => {
+  if (range.length === 0) return [0, steps.length - 1]
+  const minStep = steps.findIndex((step: number) => step === range[0])
+  const maxStep = steps.findIndex((step: number) => step === range[1])
+  return [minStep, maxStep]
+}
+
+const getRealValue = (steps: number[], value: number): number => {
+  return steps[value]
+}
+
+const StepSlider: FC<StepSliderProps> = ({
+  tree,
+  node,
+  formatter = toNumber,
+}) => {
+  const formatValue = (value: number): string => {
+    const realValue = getRealValue(steps, value)
+    const lastStepValue = steps[steps.length - 1]
+    if (realValue === lastStepValue) return `${formatter(realValue)}+`
+    return formatter(realValue).toString()
+  }
+
+  const steps = node.steps
+  const count = node.context.recordsCount ?? 0
+  const defaultRange = getDefaultRange(steps, node.range)
+  const [rangeValues, setRangeValues] = useState(defaultRange)
+
+  const onRangeUpdated = (range: number[]) => {
+    const realRange = [
+      getRealValue(steps, range[0]),
+      getRealValue(steps, range[1]),
+    ]
+    tree.mutate(node.path, { range: realRange })
+  }
+
+  const realFormattedMin = formatValue(rangeValues[0])
+  const realFormattedMax = formatValue(rangeValues[1])
+  const rangeLabel = `${realFormattedMin} -  ${realFormattedMax}`
+  const currentCount = formatter(count).toString()
+
+  return (
+    <Box>
+      <HStack paddingX="4px" justifyContent="space-between">
+        <Label value={rangeLabel} label="Range" />
+        <Label value={currentCount} label="Results" alignment="right" />
+      </HStack>
+      {/** This padding exists because of contexture native margins.
+       * When the slider thumb is encapsulated on the filter UI,
+       * part of it gets cut if no padding is applied. */}
+      <Box marginTop="6px" paddingX="11px">
```

Comment:
A request for em/rem over px

Reply:
What do you think about properly discussing that before moving forward with this change? My reason for that consideration is because we are using pixels as our measure everywhere on `contexture` and `spark`.

Doing this change in isolation wouldn't make sense if we are not willing to set that as our new standard from here and beyond. Other than that, that are other considerations we should do before starting using `rem` on a codebase founded in pixels.

Reply:
What about `em`? What are the repercussions of using that?
I might be missing something, but I don't see drawbacks specifically with em(?).

Also happy to discuss as a team. In fact, that could be more effective to tackle as a group,
v.s a siloed PR. 👍🏾